examples/sms/chicken.c

```diff
@@ -94,12 +94,15 @@ void main() {
 		while (y < 176) {
 			while (get_vcount() < y) {
 			}
+#if 0
+			// load_palette() is too slow for this to work
```

Comment:
Due to adding calls to disable/save and restore interrupts the timing of load_palette and scroll_bkg has changed. They not take longer. This appears to be too long for this example to work. By removing the palette changes at least the cars move left and right correctly.